landlab/components/radiation/radiation.py

```diff
@@ -12,15 +13,33 @@ def _assert_method_is_valid(method):
 
 class Radiation(Component):
```

Comment:
The ``Radiation`` component is missing a ``run_one_step`` method, which most *Landlab* components have.

Maybe something like the following would work?
```python
def run_one_step(dt=None):
    if dt is None:
        dt = 1.0 / 365.0
    self.current_time += dt
    self.update()
```